.gitignore

```diff
@@ -1,12 +1,13 @@
-/lib
+lib/
+.turbo/
+node_modules/
 /bin
 /web.config
 .idea
 .history
 .DS_Store
 build
 aspnet_client
-node_modules
```

Comment:
Why remove it?

Reply:
added here to recursively ignore node_modules in sub directories
https://github.com/cocos/cocos-engine/blob/55693df4c4116a76af0ba29b83d45d7f89677e31/.gitignore#L3